files/build_templates/sonic_debian_extension.j2

```diff
@@ -401,6 +409,25 @@ sudo LANG=C chroot $FILESYSTEM_ROOT docker $SONIC_NATIVE_DOCKERD_FOR_DOCKERFS ta
 sudo LANG=C chroot $FILESYSTEM_ROOT docker $SONIC_NATIVE_DOCKERD_FOR_DOCKERFS tag {{imagename}}:latest {{imagebasename}}:latest
 {% endif %}
 {% endfor %}
+
+            
+{% if install_kubernetes == "y" %}
+## Pull in kubernetes docker images
+echo "pulling universal k8s images ..."
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT docker pull k8s.gcr.io/pause:3.2
```

Comment:
>3.2 [](start = 90, length = 3)

How do you find the version strings in this section? What if we need to upgrade in future? Should we follow some instructions to find another set of versions?

Reply:
Let me make it configurable with defaults. I don't see a generic way to get a mapping  as kubernetes,, k8s.gcr.io & calico are not directly connected. 

For now, the tests we write is the perfect way to keep them in sync,

Reply:
done. Expressed in rules/config.